build/VSL.Imports.Closed.targets

```diff
@@ -70,8 +70,8 @@
           Inputs="@(IntermediateAssembly)"
           Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
     
-    <Exec Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
-    
+    <Exec Condition=" '$(OS)' == 'Windows_NT'" Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
+    <Exec Condition=" '$(OS)' != 'Windows_NT'" Command="mono $(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
```

Comment:
Is there a way we could check for mono here?  I would like to show a more user friendly message if mono is missing.